modules/pages/client/components/Foundation.component.js

```diff
@@ -0,0 +1,151 @@
+import React from 'react';
+import ManifestoText from './ManifestoText.component.js';
+import { useTranslation } from 'react-i18next';
+import '@/config/client/i18n';
+import Board from '@/modules/core/client/components/Board.js';
+
+export default function Foundation() {
+  const { t } = useTranslation('pages');
+
+  // mock data for testing
+  const app = {
+    user: true,
+  };
```

Comment:
Should this mock data be here, or is it temporary?
I see the button `Join Trustroots` is missing on `/foundation` page when user is not logged in. I suppose it's because this value is fixed. Is it meant to work similar like passing `user` prop to `Home` component?

Reply:
thanks, resolved in b3ef25cf